app/scheduler/modules/pairing.py

```diff
@@ -0,0 +1,63 @@
+"""Match two Launchpad member for a private conversation"""
```

Comment:
```suggestion
"""Match two Launch Pad members for a private conversation"""
```

😢 https://docs.ubclaunchpad.com/handbook/design/branding#launch-pad